### ISSUE TITLE
Missing sync icons in windows explorer (and also Gnome)

### ISSUE THREAD

**Issue by mathiasconradt** [nextcloud/desktop#613]
User reported that the Win10 clients in the company don't show the sync icons of the Nextcloud folder. They have users on Ubuntu, Biolinux 8.0.7 but most users on Windows 10. They also have Dropbox and OneDrive installed, where the icons show.

I experience the same. On my Ubuntu Gnome the icons don't show (for Dropbox they do though), on Bjoern's Debian Gnome the icons show just fine.

Related:
https://help.nextcloud.com/t/missing-sync-icons-in-windows-explorer/6610

![image](https://user-images.githubusercontent.com/295122/44910605-1787be00-ad24-11e8-9957-3df6b9d9fb4a.png)

**Comment by splitt3r** [comment]
Maybe a stupid question but is `nextcloud-client-nautilus` installed? I think it should be way more present in the desktop client. Didn´t know about it for too long.

**Comment by mathiasconradt** [comment]
@camilasan no, it's the stable version that one can download and install from our website https://nextcloud.com/install, or in my case on Linux via ppa.

**Comment by camilasan** [comment]
So it is definitely fixed in the current daily builds. But I will leave the issue open until we can mark one as stable and it to the website.

**Comment by mj-mehdizadeh** [comment]
i have same problem and folder sync icon is broken
i created a branch from master and i fetch the last commit of master
i am here [#cb69944](https://github.com/nextcloud/desktop/commit/cb69944b5c09a26f86ec38d7ce9a589cac950f7a)
